cmds/core/brctl/brctl_test.go

```diff
@@ -0,0 +1,217 @@
+			forceVM: true,
+		},
+		{
+			name:   "delif_fewArgs",
+			argv:   []string{"delif", "bridge0", "eth0", "eth1"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "show",
+			argv:   []string{"show"},
+			expErr: nil,
+		},
+		{
+			name:   "showmacs",
+			argv:   []string{"showmacs", "eth0"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "showmacs_fewArgs",
+			argv:   []string{"showmacs", "eth0", "eth1"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setageingtime",
+			argv:   []string{"setageingtime", "bridge0", "10ms"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setageingtime",
+			argv:   []string{"setageingtime", "bridge0", "10ms", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "stp",
+			argv:   []string{"stp", "bridge0", "10ms"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "stp_fewArgs",
+			argv:   []string{"stp", "bridge0", "10ms", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setbridgeprio",
+			argv:   []string{"setbridgeprio", "bridge0", "10"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setbridgeprio_fewArgs",
+			argv:   []string{"setbridgeprio", "bridge0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setfd",
+			argv:   []string{"setfd", "bridge0", "10ms"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setfd_fewArgs",
+			argv:   []string{"setfd", "bridge0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "sethello",
+			argv:   []string{"sethello", "bridge0", "10ms"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "sethello_fewArgs",
+			argv:   []string{"sethello", "bridge0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setmaxage",
+			argv:   []string{"setmaxage", "bridge0", "10ms"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setmaxage_fewArgs",
+			argv:   []string{"setmaxage", "bridge0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setpathcost",
+			argv:   []string{"setpathcost", "bridge0", "eth0", "10"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setpathcost_fewArgs",
+			argv:   []string{"setpathcost", "bridge0", "eth0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "setportprio",
+			argv:   []string{"setportprio", "bridge0", "eth0", "10"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "setportprio_fewArgs",
+			argv:   []string{"setportprio", "bridge0", "eth0", "10", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "hairpin",
+			argv:   []string{"hairpin", "bridge0", "eth0", "enable"},
+			expErr: os.ErrNotExist,
+		},
+		{
+			name:   "hairpin_fewArgs",
+			argv:   []string{"hairpin", "bridge0", "eth0", "enable", "garbage"},
+			expErr: errFewArgs,
+		},
+		{
+			name:   "garbage_command",
+			argv:   []string{"garbage_command"},
+			expErr: errInvalidCmd,
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.skipVM {
+				guest.SkipIfInVM(t)
+			}
+
+			if tt.forceVM {
+				guest.SkipIfNotInVM(t)
+			}
```

Comment:
I was taking a closer look at these two functions at https://github.com/hugelgupf/vmtest/blob/main/guest/skip.go and found that these two functions are identical?

```
// SkipIfNotInVM skips the test if it is not running in a vmtest-started VM.
//
// The presence of VMTEST_IN_GUEST=1 env var (which can be passed on the
// kernel commandline, using qemu.WithVmtestIdent) is used to determine this.
func SkipIfNotInVM(t testing.TB) {
	if os.Getenv("VMTEST_IN_GUEST") != "1" {
		t.Skip("Skipping test -- must be run inside vmtest VM")
	}
}

// SkipIfInVM skips the test if it is running in a vmtest-started VM.
//
// The presence of VMTEST_IN_GUEST=1 env var (which can be passed on the
// kernel commandline, using qemu.WithVmtestIdent) is used to determine this.
func SkipIfInVM(t testing.TB) {
	if os.Getenv("VMTEST_IN_GUEST") != "1" {
		t.Skip("Skipping test -- must be run inside vmtest VM")
	}
}
```

Might be a typo on the definition for `SkipIfInVM`. Can we verify that it's skipVM and forceVM is working as expected? Or submit a PR to vmtest/guest if not.

Reply:
Sure, can take a look

Reply:
https://github.com/hugelgupf/vmtest/pull/126

Lets see what @hugelgupf says :)

Reply:
I can see why it may look same. Opened to fork off the discussion, https://github.com/hugelgupf/vmtest/issues/127